src/inference/hmc.jl

```diff
@@ -39,6 +39,47 @@ end
 
 DynamicPPL.getlogp(t::HMCTransition) = t.lp
 
+###
+### Default options
+###
```

Comment:
Wouldn't it make sense to add the default options to AdvancedHMC?

Reply:
Yeah, I think that would make for a cleaner design in the end.